edit_flow.php

```diff
@@ -1,31 +1,17 @@
 <?php
-/*
-Plugin Name: Edit Flow
-Plugin URI: http://editflow.org/
-Description: Remixing the WordPress admin for better editorial workflow options.
-Author: Daniel Bachhuber, Scott Bressler, Mohammad Jangda, Automattic, and others
-Version: 0.9.8
-Author URI: http://editflow.org/
-
-Copyright 2009-2019 Mohammad Jangda, Daniel Bachhuber, Automattic, et al.
-
-GNU General Public License, Free Software Foundation <http://creativecommons.org/licenses/GPL/2.0/>
-
-This program is free software; you can redistribute it and/or modify
-it under the terms of the GNU General Public License as published by
-the Free Software Foundation; either version 2 of the License, or
-(at your option) any later version.
-
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU General Public License for more details.
-
-You should have received a copy of the GNU General Public License
-along with this program; if not, write to the Free Software
-Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-
-*/
+/**
+ * Plugin Name: Edit Flow
+ * Plugin URI: http://editflow.org/
+ * Description: Remixing the WordPress admin for better editorial workflow options.
+ * Author: Daniel Bachhuber, Scott Bressler, Mohammad Jangda, Automattic, and others
+ * Version: 0.9.8
+ * Requires at least: 6.0
+ * Requires PHP: 8.0
+ * License: GPL-3
+ * License URI: https://www.gnu.org/licenses/gpl-3.0.html
```

Comment:
Was the license change intentional?

`composer.json` still says GPL v2 or later. What is it?

I'd recommend adding a proper LICENSE file to the repo.

Reply:
Thanks for catching that. We meant to make it GPL 3 as some of our other plugins have that already, so we were trying to make it consistent. I'll fix that inconsistency, and add a proper LICENSE file to the repo.